### DISCUSSION TITLE
What is the code for "Business Owners" type?

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1360] [Q&A] [answered]
I am trying to update the description for the Business owner" type, but I cannot find the code for that type in the list of meeting codes https://github.com/code4recovery/spec#meeting-types

I am guessing it was added at some point and not included in the list.

Does anyone know the code so I can update the description?

Thank you so much!

**Comment by joshreisner** [comment, marked as answer]
that spec is currently just for AA - looks like the "Business Owner" type you're referring to for Debtors Anonymous is `B`

you can see the complete list here: https://github.com/code4recovery/12-step-meeting-list/blob/main/includes/variables.php#L925-L942

**Reply by willinrecovery** [reply]
Thank you, Josh. Yes, this is for a Debtors Anonymous site. I am guessing those codes activated when I switched to Debtors Anonymous in settings.  Thank you so much!
